app/workers/evss/disability_compensation_form/form8940_document.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module EVSS
+  module DisabilityCompensationForm
+    class Form8940Document < EvssDocument
+      FORM_ID = '21-8940' # form id for PTSD IU
+      DOC_TYPE = 'L149'
+
+      def initialize(submission)
+        parsed_form = JSON.parse(submission.form_to_json(Form526Submission::FORM_8940))
```

Comment:
there's also a `form` method on the submission from which you can get the parsed json. e.g.
```ruby
submission.form[Form526Submission::FORM_8940]
```
`form_to_json` is a helper which does the above (grabs a sub form by key) and then converts it to json for submission.